blosc2/core.py

```diff
@@ -1275,23 +1275,23 @@ def compute_chunks_blocks(
             blocks = [blocks]
         if len(blocks) != len(shape):
             raise ValueError("blocks should have the same length than shape")
-        for i in range(len(blocks)):
-            if blocks[i] == 0:
+        for b, s in zip(blocks, shape):
```

Comment:
Please use significant varnames

Reply:
I was unable to find good names. Perhaps `array_len`, `block_len`, `chunk_len`?

Reply:
What about adopting the singular case?  e.g. `dim`,  `block`, `chunk` ?

Reply:
I have applied your suggestion. Alternatives:
* `dim`, `block_dim`, `chunk_dim`
* `array_dim`, `block_dim`, `chunk_dim`